ripe/atlas/tools/probes/__init__.py

```diff
@@ -11,10 +11,6 @@ class Probe(object):
 
     EXPIRE_TIME = 60 * 60 * 24 * 30
 
-    def __init__(self, id, meta_data=None):
-
-        return CProbe(id, meta_data)
-
```

Comment:
Why? It's useful if you want to get probe without any cache! It's free, so I would let it live there